src/components/CellContainer.js

```diff
@@ -51,10 +51,7 @@ const ComposedCellContainer = OriginalComponent => compose(
     ...props,
     style: getCellStyles(props.cellProperties, props.style),
     value: props.customComponent ?
-      <props.customComponent
-        value={props.value}
-        griddleKey={props.griddleKey}
-      /> :
+      <props.customComponent {...props} /> :
```

Comment:
Another fix not specific to TS, in imitation of existing code in `TableHeadingCellContainer`.